synapse/storage/databases/main/group_server.py

```diff
@@ -750,7 +768,7 @@ def _add_room_to_summary_txn(
                 WHERE group_id = ? AND category_id = ?
             """
             txn.execute(sql, (group_id, category_id))
-            (order,) = txn.fetchone()
+            (order,) = cast(Tuple[Optional[int]], txn.fetchone())
```

Comment:
Is this correct?

- `MAX(X)` might return null if there are no `room_orders` selected,
- but if so, `COALESCE(NULL, 0)` will give us a nonnull value (`0`).

So think `order: int` rather than `order: Optional[int]`.

Reply:
I have done this.